pages/about.js

```diff
@@ -0,0 +1,82 @@
+import Image from "next/image";
+import { Button, Col, Container, Row } from "react-bootstrap";
+import { FaRegHandshake, FaRegLightbulb } from "react-icons/fa";
+import styles from '/styles/Mission.module.css';
+import Clubs from '../assets/data/clubs.json';
+
+function Affiliate(props) {
+  const { id, logo, name } = props
+  return (
+    <a href={`affiliates.html#${id}`} className={styles["affiliate-icon"]}>
+      <Image
+        src={logo}
+        alt={`${name} Logo`}
+        height="100%"
+        width="100%"
+      />
+    </a>
+  )
+}
+
+export default function Mission() {
+  return (
+    <>
+        <h1>What is ISCSC?</h1>
+        <p>ICSSC is the official student government for the Donald Bren School of Information & Computer Science (ICS) at the University of California, Irvine. We and our member clubs serve to all students who are interested in the ICS field. We welcome students of all backgrounds and skill levels to join our community and share our love for technology!
+            <br></br><br></br>
```

Comment:
Usually I like to use CSS to create space instead of multiple `<br>`s. But that can be done later when the page gets styled